csvcubed-pmd/tests/unit/csvwcli/test_pull.py

```diff
@@ -74,5 +79,14 @@ def test_get_relative_dependencies_excludes_absolute_dependencies():
     assert len(dependencies) == 0
 
 
+@pytest.mark.vcr
+def test_get_tableschema_dependencies():
+    dependencies = _get_csvw_dependencies_relative(
+        "https://ci.ukstats.dev/job/GSS_data/job/Trade/job/csvcubed/job/HMRC-alcohol-bulletin/job/HMRC-alcohol-bulletin/lastSuccessfulBuild/artifact/outputs/alcohol-sub-type.csv-metadata.json"
```

Comment:
This URL will cease to work at some point in the future. I presume that's why you've added VCR on here?

Reply:
yeah exactly, though i prefer fixtures for all urls (where sensible), quite happy with that pyest plugin tbh, nice and simple.